public/pages/CreateRollup/utils/constants.ts

```diff
@@ -107,6 +107,24 @@ export const testRollup = {
       },
     ],
   },
+  metadata: {
```

Comment:
Are these used in the source code somewhere or just tests? If only tests they should not be mixed in the source code files.

Reply:
Only used in tests, so I moved these 2 constants to `test/testConstants.ts` file.

Reply:
Renamed to `test/constants.ts`